spec/components/works/contact_email_component_spec.rb

```diff
@@ -7,7 +7,7 @@
   let(:work) { build_stubbed(:work) }
   let(:work_form) { WorkForm.new(work) }
   let(:form) { ActionView::Helpers::FormBuilder.new('work', work_form, controller.view_context, {}) }
-  let(:rendered) { render_inline(described_class.new(form: form)) }
+  let(:rendered) { render_inline(described_class.new(form: form, key: 'work.contact_email')) }
```

Comment:
It'd be great if a test here failed if the key above didn't exist if my observation above is true.

Reply:
It doesn't fail, because the missing key is gracefully handled.